<!-- move base -->
<launch>
  <master auto="start"/>

  <arg name="robot_namespace"/>


  <param name="use_sim_time" value="true" />

  <node pkg="gmapping" type="slam_gmapping" name="slam_gmapping" output="screen" >
    <remap from="scan" to="scan"/>
    <param name="map_frame" value="$(arg robot_namespace)/map"/>
    <param name="odom_frame" value="$(arg robot_namespace)/odom"/>
    <param name="base_frame" value="$(arg robot_namespace)/base_footprint"/>
    <param name="map_update_interval" value="1.0"/>
    <param name="maxUrange" value="4.0"/>
    <param name="minimumScore" value="50"/>
    <param name="linearUpdate" value="0.2"/>
    <param name="angularUpdate" value="0.2"/>
    <param name="temporalUpdate" value="0.5"/>
    <param name="delta" value="0.1"/>
    <param name="lskip" value="0"/>
    <param name="particles" value="30"/>
    <param name="sigma" value="0.05"/>
    <param name="kernelSize" value="1"/>
    <param name="lstep" value="0.05"/>
    <param name="astep" value="0.05"/>
    <param name="iterations" value="5"/>
    <param name="lsigma" value="0.075"/>
    <param name="ogain" value="3.0"/>
    <param name="srr" value="0.01"/>
    <param name="srt" value="0.02"/>
    <param name="str" value="0.01"/>
    <param name="stt" value="0.02"/>
    <param name="resampleThreshold" value="0.5"/>
    <param name="xmin" value="-8.0"/>
    <param name="ymin" value="-8.0"/>
    <param name="xmax" value="8.0"/>
    <param name="ymax" value="8.0"/>
    <param name="llsamplerange" value="0.01"/>
    <param name="llsamplestep" value="0.01"/>
    <param name="lasamplerange" value="0.005"/>
    <param name="lasamplestep" value="0.005"/>
  </node>

</launch>
